execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -453,6 +455,53 @@ impl TransactionGenerator {
         }
     }
 
+    /// Generates random P2P transfer transactions, with `1-hotspot_probability` of the accounts used `hotspot_probability` of the time.
+    ///
+    /// Example 1. If `hotspot_probability` is 0.5, all accounts have the same probability of being sampled.
+    ///
+    /// Example 2. Say there are 10 accounts A0, ..., A9 and `hotspot_probability` is 0.8.
+    /// Whenever we need to sample an account, with probability 0.8 we sample from {A0, A1} uniformly at random;
+    /// with probability 0.2 we sample from {A2, ..., A9} uniformly at random.
+    pub fn gen_random_transfers_with_hotspot(
+        &mut self,
+        block_size: usize,
+        num_blocks: usize,
+        hotspot_probability: f32,
+    ) {
+        assert!((0.5..1.0).contains(&hotspot_probability));
+        println!("Started generating random transfers with hotspot.");
```

Comment:
Remove this or change it to use logger instead.

Reply:
fixed